spice/tests/vm_listen.py

```diff
@@ -60,21 +60,17 @@ def run(vt_test, test_params, env):
                 s_port = env.get_vm(vmname).spice_port
                 error_s = error_s % (test_params['spice_addr'], s_port)
             if error_s in emsg.output and emsg.status == 1:
-                logging.info("Guest terminated as expected: %s" % emsg.output)
+                logging.info("Guest terminated as expected: %s", emsg.output)
                 return
             else:
                 raise error.TestFail("Guest creation failed, bad error message:"
                                      " %s and/or exit status: %s" %
                                      (emsg.output, emsg.status))
         finally:
-            try:
-                process.safe_kill(nc_process_pid, signal.SIGKILL)
-            except:
-                pass
+            process.safe_kill(nc_process_pid, signal.SIGKILL)
```

Comment:
try..except block was included only in case that some error emerges during `nc` process kill or if `nc` process is not present at all. I noticed, that similar try..except block is already present in `safe_kill` procedure (https://github.com/avocado-framework/avocado/blame/master/avocado/utils/process.py#L139), so there is no need for this block. It is in `finally` clause, so it is run everytime - even if we get out of `run` function on L64.